<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="48874" started-at="2020-09-06T23:34:23 BDT" finished-at="2020-09-06T23:35:12 BDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="48874" started-at="2020-09-06T23:34:23 BDT" finished-at="2020-09-06T23:35:12 BDT">
      <class name="com.login.LoginPageTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.login.LoginPageTest@436e852b]" name="beforeTest" is-config="true" duration-ms="294" started-at="2020-09-06T23:34:23 BDT" finished-at="2020-09-06T23:34:23 BDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.login.LoginPageTest@436e852b]" name="beforeMethod" is-config="true" duration-ms="17796" started-at="2020-09-06T23:34:23 BDT" finished-at="2020-09-06T23:34:41 BDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="invalidTest(java.lang.String, java.lang.String)[pri:0, instance:com.login.LoginPageTest@436e852b]" name="invalidTest" duration-ms="7656" started-at="2020-09-06T23:34:41 BDT" data-provider="InvalidData" finished-at="2020-09-06T23:34:49 BDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[example@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Warning: No match for E-Mail Address and/or Password.] but found [Warning: No match for E-Mail Address]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Warning: No match for E-Mail Address and/or Password.] but found [Warning: No match for E-Mail Address]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.login.LoginPageTest.invalidTest(LoginPageTest.java:157)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.login.LoginPageTest@436e852b]" name="tearDown" is-config="true" duration-ms="260" started-at="2020-09-06T23:34:49 BDT" finished-at="2020-09-06T23:34:49 BDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=invalidTest status=FAILURE method=LoginPageTest.invalidTest(java.lang.String, java.lang.String)[pri:0, instance:com.login.LoginPageTest@436e852b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.login.LoginPageTest@436e852b]" name="beforeMethod" is-config="true" duration-ms="5981" started-at="2020-09-06T23:34:49 BDT" finished-at="2020-09-06T23:34:55 BDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="validTest(java.lang.String, java.lang.String)[pri:0, instance:com.login.LoginPageTest@436e852b]" name="validTest" duration-ms="13868" started-at="2020-09-06T23:34:55 BDT" data-provider="ValidData" finished-at="2020-09-06T23:35:09 BDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[kafiraj.27@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[120862ks]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.login.LoginPageTest@436e852b]" name="tearDown" is-config="true" duration-ms="278" started-at="2020-09-06T23:35:09 BDT" finished-at="2020-09-06T23:35:09 BDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validTest status=SUCCESS method=LoginPageTest.validTest(java.lang.String, java.lang.String)[pri:0, instance:com.login.LoginPageTest@436e852b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.login.LoginPageTest@436e852b]" name="afterTest" is-config="true" duration-ms="2651" started-at="2020-09-06T23:35:09 BDT" finished-at="2020-09-06T23:35:12 BDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.login.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
